mention that tiny files are not compressed, also other typos (#1357)
minio/docs: source/administration/object-management/data-compression.rst

Excluded File Types
~~~~~~~~~~~~~~~~~~~

> Some data cannot be effectively compressed, such as video, already compressed data, or very small files.

Reviewer: ```suggestion
Some data cannot be effectively compressed.
For example: video, already compressed data, or very small files.
```

A suggestion to consider to avoid a dizzying amount of commas.

Author: oh yes, that is nicer 👍🏻

Excluded File Types
~~~~~~~~~~~~~~~~~~~

Some data cannot be effectively compressed.

> For example: video, already compressed data, or very small files.

Reviewer 2: ```suggestion
For example: video, already compressed data, or files less than 4KiB.
```

Author: 👍🏻 will add manually to reduce merge nonsense
